src/transaction.js

```diff
@@ -6,6 +6,20 @@ var opcodes = require('./opcodes.json')
 var typeforce = require('typeforce')
 var types = require('./types')
 
+function varSliceSize (someScript) {
+  var length = someScript.length
+
+  return bufferutils.varIntSize(length) + length
+}
+
+function vectorSize (someVector) {
+  var length = someVector.length
+
+  return bufferutils.varIntSize(length) + someVector.reduce(function (sum, witness) {
+    return sum + varSliceSize(witness)
+  }, 0)
+}
+
```

Comment:
@afk11 I renamed `varString` to `varSlice`, since that was more consistent with the rest.